### ISSUE TITLE
paper-autogrow-textarea+paper-char-counter is broken in chrome because chrome counts new lines as two characters

### ISSUE THREAD

**Issue by RobertLowe** [Polymer/paper-input#160]
Steps to reproduce:

Correctly counted: 

1. Go to: https://www.polymer-project.org/components/paper-input/demo.html
2. Type into "autogrowing text area with character counter": `aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa`

100/100 highlights/errors 'red'

Incorrectly counted: 

1. Go to: https://www.polymer-project.org/components/paper-input/demo.html
2. Type into "autogrowing text area with character counter": `aaaaaaaaaa`
3. Type: enter
4. Fill the remainder with `a`'s

99/100 error state never triggered.

The root cause is chrome's implementation counts new lines as two characters, other browsers don't.

**Comment by notwaldorf** [comment]
Yeah, I noticed this when I was testing it. I'm not sure what the best solution would be here. Always checking whether a `\n` is present and manually adjusting the counter seems messy. Maybe @morethanreal has an idea.

**Comment by RobertLowe** [comment]
@notwaldorf totally, it's messy either way; either we adjust the maxlength for chrome per new line to compensate for [this issue](https://code.google.com/p/chromium/issues/detail?id=252613) or we compensate the counter by +2 for each \n in chrome.

hopefully someone has a better idea!

**Comment by paxtontech** [comment]
Maybe something this?

    this._charCount = e.target.value.replace(/(\r\n|\n|\r)/g, '--').length;
